ocf_data_sampler/config/model.py

```diff
@@ -102,6 +102,39 @@ class TimeResolutionMixin(Base):
     )
 
 
+class Site(DataSourceMixin, TimeResolutionMixin, DropoutMixin):
+    """Site configuration model"""
+
+    filename: str = Field(
+        ...,
+        description="The NetCDF files holding the power timeseries.",
+    )
+    metadata_filename: str = Field(
```

Comment:
```suggestion
    file_path: str = Field(
        ...,
        description="The NetCDF files holding the power timeseries.",
    )
    metadata_file_path: str = Field(
```
Can do something like this for consistency maybe? Or even netcdf_path and metadata_csv_path if we've settled on pv file schema (do we think it's the same for wind btw?)

Reply:
Yep the schema should be generation type agnostic, agree "path" is a bit clearer

Reply:
thankd